week-4/Homework/mandatory/2-writers.js

```diff
@@ -53,19 +53,36 @@ Exercise 1:
   insert corresponding values to the place holder that are indicated in courle braces:
   "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
-
-
+console.log(
+  writers.map(
+    (p) =>
+      `Hi, my name is ${p.firstName}  ${p.lastName}. I am ${p.age} years old, and work as a ${p.occupation}.`
+  )
+);
 
 /*
 Exercise 2:
   Only `console.log()` the writers who are in their 40s with us anymore with the sentence:(meaning between 40 and 49)
   "Writer {firstName} {lastName} died at {age} years old."
 */
-
-
+console.log(
+  writers
+    .filter((age) => age.age < 49 && age.age > 40 && age.alive === false)
+    .map(
+      (p) => `Writer ${p.firstName} ${p.lastName} dies at ${p.age} years old.`
+    )
+);
 

```

Comment:
same here for the map, 
for the filter
alive ===false is a bit lengthy. 
whenever you are comparing a variable that is a boolean, you can use it directly
like
```!alive```